servers/src/main/java/tachyon/master/lineage/checkpoint/CheckpointLatestPlanner.java

```diff
@@ -15,31 +15,49 @@
 
 package tachyon.master.lineage.checkpoint;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import com.google.common.collect.Lists;
 
+import tachyon.Constants;
+import tachyon.exception.FileDoesNotExistException;
+import tachyon.master.file.meta.FileSystemMasterView;
 import tachyon.master.lineage.meta.Lineage;
+import tachyon.master.lineage.meta.LineageStateUtils;
 import tachyon.master.lineage.meta.LineageStoreView;
 
 /**
  * This class tries to checkpoint the latest created lineage that is ready for persistence. This
- * class serves as an example to implement a scheduler.
+ * class serves as an example to implement a planner.
  */
-public final class CheckpointLatestScheduler implements CheckpointScheduler {
+public final class CheckpointLatestPlanner implements CheckpointPlanner {
+  private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
 
   /**
-   * CheckpointLatestScheduler does not use the lineage store view.
+   * {@linkplain CheckpointLatestPlanner} does not use the lineage store view.
```

Comment:
what is the difference between `linkplain` and `link` ... I don't recall ever seeing `linkplain` elsewhere so I wonder what our convention is